### PR TITLE
Feature/bphh 883/jurisdiction list

### PR DESCRIPTION
**Pull request by bkmaibach**
## Description

Add a jurisdiction page available everywhere, even when logged out.
Add regional district name to jurisdiction search data.
add regional district and address to jurisdiction creation.
Fix last_signed_in_at search field for users
Fix name clash for jurisdiction search concern methods (order, query, etc)
Add locality type filter to jurisdiction select search

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-883
https://hous-bssb.atlassian.net/browse/BPHH-969

## Steps to QA

log in as super admin, create a new jurisdiction. provide a regional district (like nanaimo)
log out. Navigate to new jurisdiciton index through menu. See the regional district and name.

### CODE REVIEW

**Review by taw27 — CHANGES_REQUESTED** [review]
Screenshot 2024-04-02 at 10 38 50 AM
I created Yokon as regional district of metro vancouver, but it doesn't show up on the table

File: app/frontend/components/domains/jurisdictions/limited-grid-header.tsx
[the remark this answers is not in the document]
Comment (bkmaibach) [reply app/frontend/components/domains/jurisdictions/limited-grid-header.tsx:1]: good call

File: app/frontend/components/domains/jurisdictions/new-jurisdiction-screen.tsx [hunk 1]
```diff
@@ -81,17 +104,66 @@ export const NewJurisdictionScreen = observer(() => {
...
+                          label={t("jurisdiction.fields.localityType")}
+                          fetchOptions={fetchLocalityTypeOptions}
+                          fieldName={"localityType"}
+                        />
+                      )}
+
+                      <Text ml={8} mt={10}>
+                        of
+                      </Text>
+                    </Flex>
```
[the remark this answers is not in the document]
Comment (bkmaibach) [reply app/frontend/components/domains/jurisdictions/new-jurisdiction-screen.tsx:129]: Not really. Theres no way to do that correctly.

[the remark this answers is not in the document]
Comment (bkmaibach) [reply app/frontend/components/domains/jurisdictions/new-jurisdiction-screen.tsx:129]: all of the names of jurisdictions will be english names.

File: app/frontend/components/domains/jurisdictions/new-jurisdiction-screen.tsx [hunk 2]
```diff
@@ -81,17 +104,66 @@ export const NewJurisdictionScreen = observer(() => {
...
                     <Box w="50%">
                       <TextFormControl label={t("jurisdiction.new.nameLabel")} fieldName={"name"} required />
                     </Box>
                   </Flex>
+
+                  <FormControl display="flex" alignItems="center">
+                    <FormLabel htmlFor="email-alerts" mb="0">
```
[the remark this answers is not in the document]
Comment (bkmaibach) [reply app/frontend/components/domains/jurisdictions/new-jurisdiction-screen.tsx:136]: good catch

File: app/frontend/components/domains/navigation/nav-bar.tsx [hunk 3]
```diff
@@ -175,7 +175,7 @@ const NavBarMenu = observer(({}: INavBarMenuProps) => {
 
   const superAdminOnlyItems = (
     <MenuGroup>
-      <NavMenuItem label={t("home.jurisdictionsTitle")} to={"/jurisdictions"} />
+      {/* <NavMenuItem label={t("home.jurisdictionsTitle")} to={"/jurisdictions"} /> */}
```
[the remark this answers is not in the document]
Comment (bkmaibach) [reply app/frontend/components/domains/navigation/nav-bar.tsx]: deleted

**Review by taw27 — APPROVED** [review]

**Pull request merged**